src/spark/v1-process-all-subjects.py

```diff
@@ -30,6 +30,7 @@ def get_delta_apen(ts): # time-series is an array
     return delta_ApEns.item()
 
 
+postgres_url="jdbc:postgresql://ip-10-0-1-31.ec2.internal:5432/speegs"
```

Comment:
appears a lot throughout. Consider consolidating into JSON, YAML, or PY file and importing from that